pkgs/desktops/gnome/apps/gnome-clocks/default.nix

```diff
@@ -52,6 +54,7 @@ stdenv.mkDerivation rec {
     gnome-desktop
     geocode-glib_2
     geoclue2
+    gst_all_1.gstreamer
```

Comment:
I would expect also some other gstreamer packages might be needed. For testing, you should be able to run with `GST_PLUGIN_SYSTEM_PATH_1_0` variable unset (to ensure you are not getting the GStreamer libraries from environment), set alarm for few seconds in the future and verify if sound plays.

Reply:
Yup still getting the error, I'll add a few packages more and see

Reply:
Ok 

```
 ++ (with gst_all_1; [
    gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav
  ])
```

does the trick.

Reply:
Since the ogg file is hardcoded, and ogg is a good format IIRC, we should not need most of these. `playbin` is in base: https://gstreamer.freedesktop.org/documentation/playback/playbin.html?gi-language=c#factory-details

Reply:
Looks like some are still needed, if I just leave the base plugins and gstreamer sound does not play, but does not crashes.
![image](https://github.com/NixOS/nixpkgs/assets/50095635/b3f9a4d8-f17d-4bb6-afa6-1d4455858141)

Reply:
Works with
```
(with gst_all_1; [ # GStreamer plugins needed for Alarms
    gstreamer gst-plugins-base gst-plugins-good
  ]);
```